test/Microsoft.AspNet.Mvc.Core.Test/OptionDescriptors/DefaultValidationProviderProviderTest.cs

```diff
@@ -52,9 +53,8 @@ public TestModelValidationProvider(ITestService service)
 
             public ITestService Service { get; private set; }
 
-            public IEnumerable<IModelValidator> GetValidators(ModelMetadata metadata)
+            public void GetValidators(ModelValidatorProviderContext context)
             {
-                throw new NotImplementedException();
```

Comment:
I think this would be indirectly ensuring that this never gets called. is this interesting? should we consider keeping this or replacing with an alternate mechanism/